dist/spec/payid-easy-checkout.txt

```diff
@@ -0,0 +1,728 @@
+     8.1.  Motivation  . . . . . . . . . . . . . . . . . . . . . . .   8
+       8.1.1.  Design Goals  . . . . . . . . . . . . . . . . . . . .   9
+     8.2.  Example Usage . . . . . . . . . . . . . . . . . . . . . .   9
+       8.2.1.  PayID Easy Checkout Initiation  . . . . . . . . . . .   9
+       8.2.2.  PayID Easy Checkout Wallet Discovery  . . . . . . . .   9
+       8.2.3.  Assemble PayID Easy Checkout URL with Query
+               Parameters  . . . . . . . . . . . . . . . . . . . . .  10
+       8.2.4.  Redirect Payer to Their Wallet  . . . . . . . . . . .  11
+       8.2.5.  Payer Confirms Payment  . . . . . . . . . . . . . . .  11
+   9.  References  . . . . . . . . . . . . . . . . . . . . . . . . .  11
+     9.1.  Normative References  . . . . . . . . . . . . . . . . . .  11
+     9.2.  Informative References  . . . . . . . . . . . . . . . . .  12
+     9.3.  URIs  . . . . . . . . . . . . . . . . . . . . . . . . . .  12
+   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .  12
+
+1.  Terminology
+
+   This protocol can be referred to as the "PayId Easy Checkout
+   Protocol".  It uses the following terminology:
+
+   o  PayID client: a client that queries a PayID server using the PayID
+      Protocol as defined in [PAYID-PROTOCOL][].
+
+
+
+
+Kramer, et al.          Expires January 24, 2021                [Page 2]
+
+Internet-Draft   Draft 1 - PayID Easy Checkout Protocol        July 2020
+
+
+   o  PayID server: the endpoint that returns payment account(s)
+      information, which conforms to the PayID Protocol.
+
+   o  Recipient: Individual or entity receiving a payment (e.g.,
+      e-commerce merchant, charity).
+
+   o  Payer: Individual or entity originating a payment to a
+      "recipient".
+
+   o  Wallet: A device or application that holds funds (may be a non-
+      custodial wallet).
+
+   o  PayID Easy Checkout URL: The URL that is the result of the PayID
+      Easy Checkout protocol; can be used to redirect a client to a
+      wallet corresponding to a particular PayID.
+
+   The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+   "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+   "OPTIONAL" in this document are to be interpreted as described in
+   [RFC2119] and [RFC9174][].
+
+2.  Introduction
+
+   The PayID Easy Checkout Protocol is a minimal protocol that allows a
+   recipient (e.g., an online merchant or a charity) to request a
+   payment from a payer using only the payer's PayID.  Implementations
+   of the protocol should require little to no server-side engineering
+   efforts, while creating an improved and uniform user experience for
+   payers.
+
+   The main focus of the Protocol is on PayID Easy Checkout Discovery,
+   which defines how a PayID client can use a PayID to retrieve a PayID
+   Easy Checkout URL which represents a resource that the payer's
+   digital wallet can use to initiate a payment to the merchant.
+
```

Comment:
Ok. So this is a little confusing. The roles here are flipped compared to that in (Verifiable) PayID protocol. The PayID client and recipient are on one side because of the reverse flow. 

I would suggest making a note explicitly for that to avoid confusion or may be a figure?

Reply:
Correct.  I'm open to changing this terminology to instead call it a "PayID Easy Checkout Client".  That way, there is no confusion amount different types of clients